demo/site/package.json

```diff
@@ -34,6 +34,7 @@
         "babel-plugin-styled-components": "^1.0.0",
         "draft-js": "^0.11.7",
         "express": "^4.0.0",
+        "file-saver": "^2.0.5",
```

Comment:
This dependency is already installed via @comet/cms-site, you don't need to add it here as well.